build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+plugins {
+    id("java-library")
+    id("com.scireum.sirius-parent") version "11.0.5"
+    id("org.sonarqube") version "3.4.0.2513"
+}
+
+apply(plugin = "com.scireum.sirius-parent")
+
+dependencies {
+    api("com.scireum:sirius-kernel:${property("sirius-kernel")}")
```

Comment:
Is this kind of a best practice naming for gradle properties? I would prefer a more descriptive name containing version, like version-sirius-kernel